internal/cmd/list.go

```diff
@@ -58,7 +58,7 @@ func listCmd() *cobra.Command {
 				block := task.CodeBlock
 				lines := block.Lines()
 				relPath, err := filepath.Rel(getCwd(), task.DocumentPath)
-				if err != nil || !isTerminal(io.Out.Fd()) {
```

Comment:
@adambabik removing the tty check for consistency. as mentioned, we'll get an absolute path here when the repo’s worktree is assigned from searching upward for `.git`. in all other cases the path is relative. i thought about fixing it upstream but it seemed ugly having to reconstruct the fs struct with a relative as opposed to an absolute path.